### PR TITLE
#122 & #169 Feature rename Foundation Core Python & Policy Decision Point Python

### PR DESCRIPTION
**Pull request by habibimoiz**
Merging in the feature branch which consists of changes found in Issues: #122 & #169. Refactoring the foundation-core-python policy-decision-point python to remove all the aiops to aissemble. 

**https://github.com/boozallen/aissemble/issues/122: Rename Foundation Core Python module with latest aiSSEMBLE landscape**

**#122 Refactoring the Foundation Core Python module; removing aiops references. #163**

- Renamed the Foundation Metadata Core (Python) package.
- Renamed the Foundation Encryption Policy (Python) package.
- Renamed the Extensions Encryption Vault Python package.
- Renamed the Extensions Data Delivery Spark (Python) package.
- Renamed all the child directories, classes, and methods.
- Corrected all the imports inside of the velocity templates to ensure a successful build of the Foundation Core Python.
- Baton migration scripts for automated migrations from legacy package names to the new aiSSEMBLE landscape.

**https://github.com/boozallen/aissemble/issues/169: Refactor Policy Decision Point (PDP) Python module with latest aiSSEMBLE landscape.**

**PR: #122 Refactoring the Foundation Core Python module; removing aiops references.**

- Refactor the PDP package.
- Refactor all the child directories, classes, and methods.
- Corrected all the imports inside the velocity templates to ensure a successful build of PDP Python.
- Baton migration scripts for automated migrations from legacy package names to the new aiSSEMBLE landscape.

### CODE REVIEW

File: foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/AiopsReferencePDPPythonMigration.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            }
+        } catch (IOException e) {
+            logger.error("Error in determining whether a 'AIOPS' renamed properties file requires a migration.");
+        }
+
+        return shouldExecute;
+    }
+
+    /**
+     * Performs the migration if the shouldExecuteOnFile() returns true.
```
Comment (aaron-gary) [inline foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/AiopsReferencePDPPythonMigration.java:72]: minor: shouldExecuteOnFile() is external to this method.  Can the description instead just say that it renames a file name?  Meaning, technically this method can be called by any other private method regardless of what shouldExecuteOnFile() returns

File: foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/AiopsReferencePDPPythonMigration.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            isRefactoredSuccessfully = file.renameTo(renamedFile);
+        } catch (Exception e) {
+            logger.error("Error in refactoring aiops-security.properties.");
+        }
+
+        return isRefactoredSuccessfully;
+    }
+
+    /**
+     * Performs the migration if the shouldExecuteOnFile() returns true.
```
Comment (aaron-gary) [inline foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/AiopsReferencePDPPythonMigration.java:93]: same comment as above.  This method can be called regardless of what shouldExecuteOnFile() returns

**Review by aaron-gary — APPROVED** [review]

File: foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/AiopsReferencePDPPythonMigration.java (hunk 1 above)
Comment (habibimoiz) [reply foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/AiopsReferencePDPPythonMigration.java]: Updated the description to better describe functionality.

File: foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/AiopsReferencePDPPythonMigration.java (hunk 2 above)
Comment (habibimoiz) [reply foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/AiopsReferencePDPPythonMigration.java]: Updated the description to better describe functionality.

**Pull request merged**